Makefile

```diff
@@ -82,18 +82,18 @@ unit-tests:
 e2e-tests: prepare-e2e-tests e2e-tests-smoke e2e-tests-cassandra e2e-tests-es e2e-tests-self-provisioned-es e2e-tests-streaming e2e-tests-examples1 e2e-tests-examples2
 
 .PHONY: prepare-e2e-tests
-prepare-e2e-tests: crd build docker push
```

Comment:
Why has this been removed? The `crd` target is idempotent and it does make sense to be part of the preparation for the e2e tests.

Reply:
CRD is added to `global-manifests` there is no need to have it in two places.